### PR TITLE
InquiryTable in separate Component file

### PR DESCRIPTION
**Pull request by mcverter**
# Task
In order to unit test, each component needs to be in a separate component file. 

[Please include a summary of the changes here]

[Link to Ticket](Insert Link Here)

[Add links to additional documentation if exists. Otherwise, delete this block]

[Insert related tickets here. Otherwise, delete this block]

# Author Tasks

## Acceptance Criteria
Please include the acceptance criteria from the ticket here or select N/A.
- [ ] N/A

### CODE REVIEW

**Review by ilhanbae — APPROVED** [review]

File: frontend/src/components/Inquiries/InquiriesTable.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        queryKey: ["inquiries"],
+        queryFn: () => InquiriesService.readInquiries(),
+    };
+}
+
+// Format date to user's timezone
+// ex. Sep 17, 2024 14:13 PM
+function formatDate(date: Date): string {
+    try {
+        // eslint-disable-next-line
```
Comment (Eugene-Polonsky) [inline frontend/src/components/Inquiries/InquiriesTable.tsx:27]: I'm curious -- what Codacy error was throwing here, and what's our reason for ignoring? Remember, when we ignore the linter, there should at least be a quick comment mentioning why.

**Review by Eugene-Polonsky — COMMENTED** [review]
Overall looks good. A couple small comments. The bigger comment is I'm not sure where the unit tests for all this code is. When can we expect those?

File: frontend/src/components/Inquiries/InquiriesTable.tsx (hunk 1 above)
Comment (mcverter) [reply frontend/src/components/Inquiries/InquiriesTable.tsx:27]: All of this was moved from @ilhanbae 's  code so maybe he can answer.

**Comment by mcverter** [comment]
> Overall looks good. A couple small comments. The bigger comment is I'm not sure where the unit tests for all this code is. When can we expect those?

I'm working on the unit tests for the AddInquiry button now. I need to isolate that component from the InquiryTable

File: frontend/src/components/Inquiries/InquiriesTable.tsx (hunk 1 above)
Comment (ilhanbae) [reply frontend/src/components/Inquiries/InquiriesTable.tsx]: Hi @Eugene-Polonsky, I believe it was something related to error catching. 
I tried to undo the eslint-disable here: https://github.com/Incredihire/Survey/pull/54

But actually it doesn't seem to show Codacy errors anymore.

Comment (mcverter) [reply frontend/src/components/Inquiries/InquiriesTable.tsx:27]: codacyError

**Comment by mcverter** [comment]
@Eugene-Polonsky @ilhanbae I tested the Codacy by changing eslint-ignore, committing, and then reverting.  I should have done this on a separate branch! Can you two re-approve this PR.  Thanks!

File: frontend/src/components/Inquiries/InquiriesTable.tsx (hunk 1 above)
Comment (ilhanbae) [reply frontend/src/components/Inquiries/InquiriesTable.tsx:27]: So, I used try catch block to handle errors, but now I see that I should've checked if each variables were valid first.

**Review by ilhanbae — APPROVED** [review]

**Comment by mcverter** [comment]
> Overall looks good. A couple small comments. The bigger comment is I'm not sure where the unit tests for all this code is. When can we expect those?

This is the work-in-progress PR  https://github.com/Incredihire/Survey/pull/55. It is branched from this PR so lmk if you can approve this one so it can be merged @Eugene-Polonsky

**Review by Eugene-Polonsky — APPROVED** [review]

**Pull request merged**